tests/pmap_test.py

```diff
@@ -1877,11 +1877,12 @@ def f(x):
     with jtu.count_jit_and_pmap_compiles() as count:  # noqa: F841
       _, f_bwd  = jax.vjp(f, x)
       _ = f_bwd(x)
-    self.assertEqual(count[0], 2)  # one for fwd, one for bwd
+    self.assertEqual(count[0], 3)  # once each for fwd, bwd, and _multi_slice
```

Comment:
`_multi_slice` (used for dispatch) now gets compiled again, despite the 'warmup' call to `f(x) ` a few lines up, because it's in a new axis env context... not sure what's up with that yet.